packages/legend-application-studio/src/stores/editor/NewElementState.ts

```diff
@@ -471,23 +477,52 @@ export class NewPackageableConnectionDriver extends NewElementDriver<Packageable
 
 export class NewServiceDriver extends NewElementDriver<Service> {
   mappingOption?: PackageableElementOption<Mapping> | undefined;
+  runtimeOption?: RuntimeOption | undefined;
```

Comment:
would argue that runtimeOption should never be undefined. we should have custom if no compatible runtimes are found

Reply:
if no compatible runtimes are found, there is an option called custom in the dropdown to proceed

Reply:
im talking about the default runTimeOption
i dont think we should have it be nullable.

Reply:
the list always have {custom} so it's not nullable. i removed '| undefined'